Source/EasyNetQ/DefaultRpc.cs

```diff
@@ -190,16 +207,21 @@ CancellationToken cancellationToken
         if (responseSubscriptions.TryGetValue(rpcKey, out var responseSubscription))
             return responseSubscription.QueueName;
 
-        logger.Debug("Subscribing for {requestType}/{responseType}", requestType, responseType);
+        logger.Debug($"Subscribing for {requestType}/{responseType}", requestType, responseType);
```

Comment:
```suggestion
        logger.Debug("Subscribing for {requestType}/{responseType}", requestType, responseType);
```

Do not inline structured logs, please.

Reply:
Fixed.